package.json

```diff
@@ -7,16 +7,18 @@
     "url": "git+https://github.com/danawoodman/react-fontawesome.git"
   },
   "config": {
-    "entry": "src/index.js"
+    "entry": "src",
+    "output": "lib",
+    "mainFile": "src/index.js"
   },
   "bugs": {
     "url": "https://github.com/danawoodman/react-fontawesome/issues"
   },
   "main": "lib/index.js",
   "scripts": {
-    "build": "babel -o $npm_package_main $npm_package_config_entry",
+    "build": "babel $npm_package_config_entry -o $npm_package_config_output",
```

Comment:
I just updated the command so it outputs to a directory instead using the `--out-dir` option.

More info: https://babeljs.io/docs/usage/cli/ (see the section titled Compiling Directories)

On my system it creates the `lib` directory if it's missing, so I don't think a `.gitkeep` file is necessary anymore.